src/controllers/ngosController.js

```diff
@@ -16,3 +17,20 @@ exports.getNgos = async (req, res, next) => {
   const ngos = await Ngo.find();
   return res.status(200).json({ success: true, data: ngos });
 };
+exports.getNgoEvents = async (req, res, next) => {
+  const ngo = await Ngo.findById(req.params.id);
+
+  if (!ngo) {
+    return next(
+      new ErrorResponse('This Id dose not match any registered NGO ', 404)
+    );
+  }
+  if (ngo.publishedEvents.length === 0) {
+    return next(
+      new ErrorResponse('This Ngo has not published any events yet ', 404)
```

Comment:
Do we need to throw error here? Because this info will be shown on NGO page and if they don't have any events the section will be empty. Is it considered as an error? @muslimomar @yamanrajab90

Reply:
yes, Good point @dilarafirtina, it is fixed now.